packages/vhd-lib/merge.js

```diff
@@ -144,6 +144,7 @@ module.exports.mergeVhd = limitConcurrency(2)(async function merge(
         parent: { header: parentVhd.header.checksum },
         currentBlock: 0,
         mergedDataSize: 0,
+        childPath,
```

Comment:
Save the whole chain as a single array, let's use our ideal format right now to avoid compatibility issues in the future.

Also, all the paths should be relative from the merge state file itself.